### PR TITLE
Bump version to 1.4.1

### PR DESCRIPTION
**Pull request by ymmt2005**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,7 +38,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 - `IsNetworkUnreachable`, `IsConnectionRefused`, `IsNoRouteToHost` functions to identify network errors.
 
-[Unreleased]: https://github.com/cybozu-go/log/compare/v1.4.0...HEAD
+[Unreleased]: https://github.com/cybozu-go/log/compare/v1.4.1...HEAD
+[1.4.1]: https://github.com/cybozu-go/log/compare/v1.4.0...v1.4.1
```
Comment (morimoto-cybozu) [inline CHANGELOG.md]: s/log/netutil/g
WTF...

Comment (ymmt2005) [reply CHANGELOG.md]: @morimoto-cybozu Thanks.  Fixed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,16 @@
-# Change Log
+# Change netutil
```
Comment (morimoto-cybozu) [inline CHANGELOG.md]: overkill?

Comment (ymmt2005) [reply CHANGELOG.md]: sigh.. fixed.

**Review by morimoto-cybozu — APPROVED** [review]
LGTM

**Pull request merged**